src/pages/wallet-new-passphrase/wallet-new-passphrase.spec.ts

```diff
@@ -204,4 +203,56 @@ describe("WalletNewPassphrasePage", () => {
     expect(comp.isPickedWord("eleven")).toBe(false);
     expect(comp.isPickedWord("three")).toBe(false);
   });
+
+  it("should call initialize in the iondidviewload", () => {
+    spyOn(comp, "initialize");
+
+    comp.ionViewDidLoad();
+
+    expect(comp.initialize).toHaveBeenCalled();
+  });
+
+  it("should call shuffleWords and set the state to enterPassphrase", () => {
+    spyOn(comp, "shuffleWords");
+
+    comp.goToEnterState();
+
+    expect(comp.shuffleWords).toHaveBeenCalled();
+    expect(comp.state).toBe("enterPassphrase");
+  });
+
+  it("should call the push method with the expected arguments", () => {
```

Comment:
Small thing: strictly speaking the WalletNewPassphrasePage does not have a push method (which this test seems to imply). The NavController does though.

Maybe rename to something more generic: `it should navigate to the WalletNewPasswordPage correctly`?